rosdep/python.yaml

```diff
@@ -2564,19 +2564,15 @@ python-mock:
     xenial_python3: [python3-mock]
     yakkety: [python-mock]
     zesty: [python-mock]
-python-more-itertools-pip:
-  arch:
-    pip:
-      packages: [more-itertools]
-  debian:
-    pip:
-      packages: [more-itertools]
-  fedora:
-    pip:
-      packages: [more-itertools]
+python-more-itertools:
+  arch: [python-more-itertools]
+  debian: [python-more-itertools]
+  fedora: [python-more-itertools]
   ubuntu:
-    pip:
-      packages: [more-itertools]
+    xenial:
+      pip:
+        packages: [more-itertools]
+    bionic: [python-more-itertools]
```

Comment:
This is failing CI because the list has to be in alphabetical order.

Reply:
Aha! I just couldn't figure out why, but this list also needs to be alphabetical. I'll fix!